./ipfs diag cmds panics without daemon running

I'm running IPFS 0.4.2. Immediately after installing I ran `./ipfs diag cmds` which paniced:

```
me@there:~/go-ipfs$ ./ipfs diag cmds
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x584857]

goroutine 1 [running]:
panic(0xe4ae40, 0xc8200100c0)
        /home/whyrusleeping/go/src/runtime/panic.go:464 +0x3e6
github.com/ipfs/go-ipfs/commands.(*ReqLog).Report(0x0, 0x0, 0x0, 0x0)
        /builds/distributions/dists/go-ipfs/gopath/src/github.com/ipfs/go-ipfs/commands/reqlog.go:108 +0x247
github.com/ipfs/go-ipfs/core/commands.glob.func1(0x767a7bae6868, 0xc8200c23c0, 0x767a7bae69f8, 0xc8200540e0)
        /builds/distributions/dists/go-ipfs/gopath/src/github.com/ipfs/go-ipfs/core/commands/active.go:22 +0x3d
github.com/ipfs/go-ipfs/commands.(*Command).Call(0x16230c0, 0x767a7bae6868, 0xc8200c23c0, 0x0, 0x0)
        /builds/distributions/dists/go-ipfs/gopath/src/github.com/ipfs/go-ipfs/commands/command.go:110 +0x472
main.callCommand(0x767a7bae6940, 0xc820016dc0, 0x767a7bae6868, 0xc8200c23c0, 0x16230c0, 0x1600320, 0x0, 0x0, 0x0, 0x0)
        /builds/distributions/dists/go-ipfs/gopath/src/github.com/ipfs/go-ipfs/cmd/ipfs/main.go:346 +0xa84
main.(*cmdInvocation).Run(0xc820016d80, 0x767a7bae6940, 0xc820016dc0, 0x0, 0x0, 0x0, 0x0)
        /builds/distributions/dists/go-ipfs/gopath/src/github.com/ipfs/go-ipfs/cmd/ipfs/main.go:189 +0x1e7
main.main()
        /builds/distributions/dists/go-ipfs/gopath/src/github.com/ipfs/go-ipfs/cmd/ipfs/main.go:154 +0x660
```

The daemon I had not started. Starting that and running the above command again yields proper output. An obvious oversight on my part, but I still think that it should not panic but return a proper error message instead.

--- a/cmd/ipfs/ipfs.go
+++ b/cmd/ipfs/ipfs.go
@@ -105,6 +105,7 @@ var cmdDetailsMap = map[*cmds.Command]cmdDetails{
 	commands.CommandsDaemonCmd:            {doesNotUseRepo: true},
 	commands.VersionCmd:                   {doesNotUseConfigAsInput: true, doesNotUseRepo: true}, // must be permitted to run before init
 	commands.LogCmd:                       {cannotRunOnClient: true},
+	commands.ActiveReqsCmd:                {cannotRunOnClient: true},
 	commands.RepoFsckCmd:                  {cannotRunOnDaemon: true},
 	commands.ConfigCmd.Subcommand("edit"): {cannotRunOnDaemon: true, doesNotUseRepo: true},
 }

